src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -172,16 +162,15 @@ def apply(self, organizations, parameter_store, config):  # pylint: disable=R091
                         )
             except ParameterNotFoundError:
                 LOGGER.debug(
-                    'Parameter %s was not found in Parameter Store, '
-                    'continuing.',
+                    "Parameter %s was not found in Parameter Store, " "continuing.",
```

Comment:
This looks wrong. If they are on the same line, it should be one string like so:

```suggestion
                    "Parameter %s was not found in Parameter Store, continuing.",
```